docs/indepth/installation.rst

```diff
@@ -18,8 +18,17 @@ It contains the services that need to run on each node.
 Installing ``flocker-cli``
 ==========================
 
-Linux
------
+Ubuntu 14.04
```

Comment:
Made it specific to Ubuntu 14.04. We don't have any idea whether the deleted instructions work for Debian either, so this is a more pragmatic solution.,